docs/dev/compiler-toolchain/solidity.md

```diff
@@ -162,31 +161,31 @@ This output format is utilized in Yul and LLVM IR compilation modes. These modes
   Output zkEVM bytecode of the contracts.  
 
 - `--debug-output-dir <path>`  
-  Dump all IRs to files in the specified directory.  
+  Dump all IR to files in the specified directory.  
   Only for testing and debugging.
 
 - `--llvm-verify-each`  
-  Set the verify-each option in LLVM.  
-  Only for testing and debugging.
+  Set the `-verify-each` option in LLVM.  
+  For testing and debugging.
```

Comment:
Why remove 'only'?

Reply:
d662925c
> and removed “Only” from “For testing and debugging” in  `-verify-each`,
since a paranoid and patient developer might want to use it more generally.